Makefile

```diff
@@ -3,9 +3,24 @@ GOINSTR += $(notdir $(patsubst %/,%,$(dir $(shell find examples/instrumentation
 
 CLOUDSDK_CONFIG?=${HOME}/.config/gcloud
 PROJECT_ID?=$(shell gcloud config get-value core/project)
-GMP_CLUSTER?=gmp-test-cluster
+PROJECT_ID:=$(if $(PROJECT_ID),$(PROJECT_ID),gmp-project)
```

Comment:
This line can just be `PROJECT_ID?=gmp-project`. The [question mark means it only proceeds if the variable is unassigned](https://www.gnu.org/software/make/manual/html_node/Setting.html#Setting-Variables), as above.

Reply:
Done